FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90

```diff
@@ -3614,7 +3268,7 @@ subroutine GFS_externaldiag_populate (ExtDiag, Model, Statein, Stateout, Sfcprop
     idx = idx + 1
     ExtDiag(idx)%axes = 2
     ExtDiag(idx)%name = 'ifd'
-    ExtDiag(idx)%desc = 'nsst index to start dtlm run or not'
+    ExtDiag(idx)%desc = 'nsst idx to start dtlm run or not'
```

Comment:
is this change intentional?

Reply:
Thanks, good catch; this was a find and replace error